packages/teleport/src/services/history.js

```diff
@@ -43,6 +43,10 @@ const history = {
     }
   },
 
+  openNewTab(route) {
+    window.open(route, '_blank');
```

Comment:
I wasn't sure if I should be checking for a return value of null (which means the window couldn't be opened for whatever reason): https://developer.mozilla.org/en-US/docs/Web/API/Window/open

I was thinking that either:
  1: It won't open
  2: Let user know that it isn't working, by popup modal

Reply:
This functionality does not really belong to history API, instead lets create a local function in that component.

```tsx
newTab(url: string) {      
    const element = document.createElement('a');
    element.setAttribute('href', `${url}`);
    // works in ie11
    element.setAttribute('target', `_blank`);
    element.style.display = 'none';
    document.body.appendChild(element);
    element.click();
    document.body.removeChild(element);
  },
```

Reply:
revised